crates/core/src/chunking.rs

```diff
@@ -15,46 +15,51 @@
 //
 // I will always be chunkified ...
 
-use std::marker::Unpin;
+use std::{marker::Unpin, time::Duration};
 
+use anyhow::{anyhow, Context};
 use async_nats::jetstream::{
     self,
     object_store::{Config, ObjectStore},
-    Context,
+    Context as JetstreamContext,
 };
 use futures::TryFutureExt;
 use tokio::io::{AsyncRead, AsyncReadExt};
 use tracing::{debug, error, instrument};
 
-use crate::error::{InvocationError, InvocationResult};
+/// Amount of time to add to rpc timeout if chunkifying
+pub const CHUNK_RPC_EXTRA_TIME: Duration = Duration::from_secs(13);
 
 /// Maximum size of a message payload before it will be chunked
 /// Nats currently uses 128kb chunk size so this should be at least 128KB
+#[cfg(not(test))]
 const CHUNK_THRESHOLD_BYTES: usize = 1024 * 900; // 900KB
+#[cfg(test)]
+const CHUNK_THRESHOLD_BYTES: usize = 1024 * 1; // 1KB
 
 /// check if message payload needs to be chunked
-pub(crate) fn needs_chunking(payload_size: usize) -> bool {
+pub fn needs_chunking(payload_size: usize) -> bool {
     payload_size > CHUNK_THRESHOLD_BYTES
 }
```

Comment:
Why is a function necessary for this, wouldn't it be much clearer and useful to just export the constant?

Reply:
I think the function call is more clear, and this was existing code, but I don't care strongly either way. I'll change callers to compare against `CHUNK_THRESHOLD_BYTES`